### DISCUSSION TITLE
Wanted: Help with setup + configuration + customisation

### DISCUSSION THREAD

**Discussion by frank-from-cologne** [jonaswinkler/paperless-ng#1427] [Support] [answered]
Hey,
ich bin ein User, kein Developer.
Ist hier jemand, die/der für mich und andere gegen angemessen Bezahlung paperless-ng-Instanz ansetzt und im Rahmen der Möglichkeiten konfiguriert?
Und vielleicht sogar ein paar Erweiterungen vornehmen kann (nice to have, ist nicht Voraussetzung)?

Hey,
I'm a user, not a developer.
Is there anyone here who can set up a paperless-ng instance for me (and others) for a reasonable fee and configure it within reason?
And maybe even make a few extensions (nice to have, not a requirement)?

Das wäre toll, That would be great
Danke im voraus, Thanks in advance

Frank (Cologne, Germany)

**Comment by TobenderZephyr** [comment, marked as answer]
Wie ist denn die genaue Voraussetzung?
Soll die Instanz im lokalen Netzwerk laufen oder aus dem Internet erreichbar sein? In beiden Fällen sind SSL-Zertifikate unabdinglich, hier kann auch auf Let's Encrypt zurückgegriffen werden.
Ist eine Domain vorhanden, um Paperless nicht per IP aufrufen zu müssen?
Ist ein Host (virtuelle Maschine oder Bare Metal) vorhanden? Reichen simple PDFs oder sollen Office Dokumente ebenfalls archiviert werden (höherer Installationsaufwand)
Wie sollen die Dokumente ins paperless-ng gelangen? Upload, Netzwerkfreigabe (nur im LAN), E-Mail?

Davon abgesehen ist die Installationsanleitung eigentlich straight-forward.
Weiteres gerne per PM

---
I was just asking a few technical details which I didn't want to translate on my mobile, sorry for that.

**Reply by frank-from-cologne** [reply]
Hey Zephyr, danke für deine schnelle Antwort.

gerne weiter per PM, mich erreichst du bei Signal 01796997040 oder per Mail frank.runge@icloud.com

Wonach ich konkret suche (in ganz kurz):

KURZFRISTIG
– 1x p-ng ist bereits aufgesetzt auf Webserver (Sub-Domain), empfängt PDFs per Mail + manuellem Upload
– Rat und Tat: Geht das und das? Kannst du das bitte konfigurieren?
– Rat und Tat: Ist eine Erweiterung möglich für dieses oder jenes (zB [Feld für Notizen](https://github.com/jonaswinkler/paperless-ng/discussions/751) )? Könntest/Würdest du das o.ä. umsetzen?

LANGRISTIG
– weitere Instanzen aufsetzen (dito auf Webservern)
– vorkonfigurieren nach Briefing
– Golden-Docker-Image für vereinfachtes roll-out vorbereiten
– … 

Danke schon Mal!
Und beste Grüße
Frank

---

> Hey Zephyr, thanks for your quick reply.
> You can reach me at Signal 01796997040 or by email frank.runge@icloud.com
> What I am specifically looking for (in very brief):

> SHORT TERM
> _- p-ng is already set up on a webserver (sub-domain), receives PDFs by mail + manual upload._
> - Advice + Hands On: Is this and this possible? Can you please configure it?
> - Advice + Hands On: Is an extension possible for this or that (e.g. [field for notes](https://github.com/jonaswinkler/paperless-ng/discussions/751)? Could/would you implement this?

> LONGER TERM
> - set up further instances (ditto on web servers)
> - pre-configure instance according to individual briefing
> - Prepare Golden Docker image for simplified roll-out
> - ...
